include/unifex/tag_invoke.hpp

```diff
@@ -16,8 +16,7 @@
 #pragma once
 
 #include <unifex/config.hpp>
-
-#include <type_traits>
```

Comment:
Still using `std::bool_constant` in this file.
Should this #include remain?

Reply:
`<type_traits>` is included by `<unifex/type_traits.hpp>`. That was my thinking anyway.